scss/2-tools/_interactive-elements.scss

```diff
@@ -1,8 +1,9 @@
 @mixin cads-interactive-focus() {
-  outline: none;
-  border-color: $cads-language__focus-border-colour;
+  outline: $cads-border-width-medium solid $cads-language__focus-colour;
```

Comment:
I had to add outline as the input value was moving slightly when the user focuses on the field. I used the gov.uk approach